assemblies/pentaho-war/src/main/webapp/jsp/PUCLogin.jsp

```diff
@@ -310,6 +310,13 @@
     evaluationPanel.toggleClass("afterSlide");
     $("#eval-arrow").toggleClass("closed");
   }
+
+  $('#eval-users-toggle').on('keydown', function(e) {
+    var keyCode = e.keyCode || e.which;
+    if (keyCode === 13 || keyCode === 32) {
+      toggleEvalPanel();
+    }
```

Comment:
Please, add a comment which translates these key codes into English language. Alternatively, create constant variables with the corresponding names.

Also, instead of calling `toggleEvalPanel()` directly, it would be best to simply call `click()` on the element. Something like:
```js
e.target.click();
```

Reply:
Added comments to translate keycodes, and replaced `toggleEvalPanel()` with `e.target.click()`.